var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -7,9 +7,10 @@
 import sys
 
 from spack import *
+from spack.build_systems.windows_variants import WindowsPackage
 
 
-class Boost(Package):
+class Boost(Package, WindowsPackage):
```

Comment:
(question) is `boost` a dependency of `hdf5`? If not IMO it should be removed from this PR.

Reply:
It doesn't appear so. Removing both the inheritance and import